smart_contracts/contracts/system/auction-install/src/main.rs

```diff
@@ -118,6 +121,20 @@ pub extern "C" fn install() {
             VALIDATOR_SLOTS_KEY.into(),
             storage::new_uref(validator_slots).into(),
         );
+        named_keys.insert(
+            ARG_AUCTION_DELAY.into(),
```

Comment:
nit: Why not have `AUCTION_DELAY_KEY` that's separate from `ARG_AUCTION_DELAY`. This way if we'd change this arguments value, we won't risk invalidating existing named keys

Reply:
Agree strongly with @mpapierski here.